app/views/admin/schedules/_day_tab.html.haml

```diff
@@ -1,18 +1,24 @@
+- cell_duration = EventType::LENGTH_STEP
+- compact_grid = EventType::LENGTH_STEP < 15
+- cells_per_hour = 60 / cell_duration
```

Comment:
same here, it seems to be more readable to me, but I may adjust it to your preference